rest/config.go

```diff
@@ -246,8 +246,9 @@ type DeprecatedCacheConfig struct {
 }
 
 type RevCacheConfig struct {
```

Comment:
This should have a corresponding change in the API specs for db config.

Reply:
Totally didn't think about this, thanks!